PhysicsTools/SelectorUtils/src/classes.h

```diff
@@ -10,14 +10,28 @@
 #include "PhysicsTools/SelectorUtils/interface/PFJetIDSelectionFunctor.h"
 #include "PhysicsTools/SelectorUtils/interface/PVSelector.h"
 #include "PhysicsTools/SelectorUtils/interface/RunLumiSelector.h"
-#include "PhysicsTools/SelectorUtils/interface/MakePyVIDClassBuilder.h"
+
+#include "PhysicsTools/SelectorUtils/interface/VersionedSelector.h"
+#include "DataFormats/PatCandidates/interface/Photon.h"
+#include "DataFormats/EgammaCandidates/interface/PhotonFwd.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
+#include "DataFormats/PatCandidates/interface/Muon.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectronFwd.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
+#include "DataFormats/PatCandidates/interface/Electron.h"
 
 namespace PhysicsTools_SelectorUtils {
   struct dictionary {
     pat::strbitset strbitset;
     edm::Wrapper<pat::strbitset> wstrbitset;
     std::vector<pat::strbitset> vstrbitset;
     edm::Wrapper<std::vector<pat::strbitset> > wvstrbitset;
+
+    VersionedSelector<edm::Ptr<reco::Photon> > vPhotonSelector;
+    VersionedSelector<edm::Ptr<pat::Photon> > vPatPhotonSelector;
+    VersionedSelector<edm::Ptr<reco::Muon> > vMuonSelector;
+    VersionedSelector<edm::Ptr<reco::GsfElectron> > vGsfElectronSelector;
+    VersionedSelector<edm::Ptr<pat::Electron> > vPatElectronSelector;
   };
```

Comment:
This struct is not needed anymore (for a long time).
```suggestion
```

Reply:
Okay, thanks for the suggestion!